Mastodon/Scene/Share/View/Content/NotificationView+Configuration.swift

```diff
@@ -154,7 +154,7 @@ extension NotificationView {
                 )
             case .status:
                 self.viewModel.notificationIndicatorText = createMetaContent(
-                    text: L10n.Scene.Notification.NotificationDescription.mentionedYou,
+                    text: .empty,
```

Comment:
Why a string-extension and not just `String()` or `""`?
(Just a question, no need to change that :) )

Reply:
I just have the feeling like this looks a bit more deliberate.